.travis.yml

```diff
@@ -34,29 +34,39 @@ matrix:
   fast_finish: true
   include:
 # 5.6
-    - php: 5.6
+    - if: type != push # disable tests on merge, run them using a cron job in the night
```

Comment:
I might be misreading the doc, but I think it should be possible [to apply the condition for the entire build](https://docs.travis-ci.com/user/conditional-builds-stages-jobs#conditional-builds).